Setting tags and layers through properties corrections (#64)
mulesoft/docs-visualizer: modules/ROOT/pages/use-tags-in-visualizer.adoc

== Setting the Tag Through Properties

To automate adding tag(s) to a node, use the property `anypoint.platform.visualizer.tags=<name>, <name>` in the application's properties file, or at deployment time through the Runtime Manager UI or API. 

[NOTE]

> You can only use the Runtime Manager UI for Cloudhub deployments.

Reviewer: cloudhub and rtf

Author: ```suggestion
You can only use the Runtime Manager UI for Cloudhub and Runtime Fabric deployments.
```

> You can only use the Runtime Manager UI for Cloudhub and Runtime Fabric deployments.

Reviewer 2: ditto previous edit questions and suggestions
Note also that if you keep "you can only use," switch it to "you can use only."

Author: That might be a touch confusing. You can set the property in CloudHub and RTF in ways besides the Runtime Manager UI. Here I'm stating Runtime Manager UI can only be used with those deployment types, not that those deployment types require that method. Does that make sense?

Author: ```suggestion
You can automate adding tags to a node by using `anypoint.platform.visualizer.tags=<name>, <name>`, either in the application's properties file or at deployment time through the Runtime Manager UI or API. 

[NOTE]
You can only use the Runtime Manager UI for Cloudhub and Runtime Fabric deployments.
```

== Setting the Tag Through Properties

> To automate adding tag(s) to a node, use the property `anypoint.platform.visualizer.tags=<name>, <name>` in the application's properties file, or at deployment time through the Runtime Manager UI or API. You can only use the Runtime Manager UI for Cloudhub deployments.

Reviewer: See previous comment: https://github.com/mulesoft/docs-visualizer/pull/64/files#r408290389

Author: ```suggestion
To automate adding tag(s) to a node, use the property `anypoint.platform.visualizer.tags=<name>, <name>` in the application's properties file, or at deployment time through the Runtime Manager UI or API. 

[NOTE]
You can only use the Runtime Manager UI for Cloudhub deployments.
```
